src/components/RadioButton.js/style.module.css

```diff
@@ -0,0 +1,63 @@
+.oc_radiobutton_container {
+    display: block;
```

Comment:
Can you do some formatting here, please?

Reply:
The styling isn't apt yet I just wanted to get the error resolved

Reply:
Alright, thanks for the clarity.